python/cudf/dataframe/series.py

```diff
@@ -1346,6 +1346,12 @@ def to_hdf(self, path_or_buf, key, *args, **kwargs):
         import cudf.io.hdf as hdf
         hdf.to_hdf(path_or_buf, key, self, *args, **kwargs)
 
+    @ioutils.doc_to_dlpack()
+    def to_dlpack(self):
+        """{docstring}"""
+        import cudf.io.dlpack as dlpack
+        return dlpack.to_dlpack(self)
```

Comment:
It strikes me with four identical `to_dlpack` methods for `column`, `index, `series`, and `dataframe` that we could use a single one line pragma to do this.

Or should this go into a `class Frame` and have them all inherit?

Reply:
This will likely eventually going into a `class Frame` but I was not willing to try to go down that rabbit hole right before 0.6 release.